adafruit_bitmap_font/bdf.py

```diff
@@ -92,6 +92,10 @@ def load_glyphs(self, code_points):
         if isinstance(code_points, int):
             remaining = set()
             remaining.add(code_points)
+        elif isinstance(code_points, str):
+            remaining = set(code_points.encode("utf-8"))
```

Comment:
Hm, I am not sure this is right.  Say you pass in the string `"ß"` which contains the Unicode code point 223.  "remaining" will not get a set containing the number 223, but instead the set containing the two numbers {195, 159} because these are the bytes in the UTF-8 encoding of ß.

I might have written instead: `remaining = set(ord(c) for c in code_points)`.

Reply:
Ah, true!